### PR TITLE
Ofir/sat 1490

### PR DESCRIPTION
**Pull request by OfirZinkerz**
AT-1513
merge to complete exercise act

### CODE REVIEW

File: src/components/filters/ellipsis.filter.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+(function (angular) {
```
Comment (assafshp) [inline src/components/filters/ellipsis.filter.js]: check cutString

File: src/components/assignModule/assignModule.service.js [hunk 2]
```diff
@@ -183,49 +183,74 @@
...
+                                }
+
+                                _summary.correctAnswersNum = _exerciseResults[exercise.exerciseTypeId][exercise.exerciseId].correctAnswersNum || 0;
+                                _summary.wrongAnswersNum = _exerciseResults[exercise.exerciseTypeId][exercise.exerciseId].wrongAnswersNum || 0;
+                                _summary.skippedAnswersNum = _exerciseResults[exercise.exerciseTypeId][exercise.exerciseId].skippedAnswersNum || 0;
+                                _summary.duration = _exerciseResults[exercise.exerciseTypeId][exercise.exerciseId].duration || 0;
+                                _summary.totalAnswered = _summary.correctAnswersNum + _summary.wrongAnswersNum;
+                            }
+
+                            // moduleSummary.overAll
```
Comment (assafshp) [inline src/components/assignModule/assignModule.service.js]: what is the meaning of this comments?

File: dist/znkExercise/locale/en.json [hunk 3]
```diff
@@ -12,19 +12,5 @@
         "STOP": "STOP",
         "PASSAGE": "Passage ",
         "SECTION": "Section"
-    },
```
Comment (assafshp) [inline dist/znkExercise/locale/en.json]: why deleted?

Comment (OfirZinkerz) [inline dist/znkExercise/locale/en.json]: it get the translation from the component it self

File: src/components/assignModule/assignModule.service.js (hunk 2 above)
Comment (OfirZinkerz) [inline src/components/assignModule/assignModule.service.js]: delete.

File: src/components/filters/ellipsis.filter.js (hunk 1 above)
Comment (OfirZinkerz) [inline src/components/filters/ellipsis.filter.js]: change to cutString

File: dist/main.js [hunk 4]
```diff
@@ -5170,6 +5188,37 @@ angular.module('znk.infra.exerciseUtility').run(['$templateCache', function($tem
 (function (angular) {
     'use strict';
     /**
+     * @returns Truncated text after the given num and add '...'
+     */
+
+    angular.module('znk.infra.filters').filter('ellipsis', function () {
```
Comment (assafshp) [inline dist/main.js]: you need to re-run grunt dist, because it was not deleted in dist

Comment (OfirZinkerz) [inline dist/main.js]: ok

**Pull request merged**